libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -554,11 +555,31 @@ void AP_DDS_Client::on_topic(uxrSession* uxr_session, uxrObjectId object_id, uin
         }
 
         if (rx_joy_topic.axes_size >= 4) {
-            GCS_SEND_TEXT(MAV_SEVERITY_INFO, "%s Received sensor_msgs/Joy: %f, %f, %f, %f",
-                          msg_prefix, rx_joy_topic.axes[0], rx_joy_topic.axes[1], rx_joy_topic.axes[2], rx_joy_topic.axes[3]);
-            // TODO implement joystick RC control to AP
+            GCS_SEND_TEXT(MAV_SEVERITY_DEBUG, "%s Received %d sensor_msgs/Joy: %f, %f, %f, %f",
```

Comment:
can't send a text msg per joy msg, would flood the link

Reply:
For context, ardupilot sends all message to the GCS by default. It's not like ROS where it prevents publish.

Reply:
done

Reply:
I removed all the GCS communications.